sdk/communication/communication-phone-numbers/src/generated/src/models/index.ts

```diff
@@ -161,10 +161,90 @@ export interface PhoneNumberSearchResult {
   searchExpiresBy: Date;
 }
 
+export interface PhoneNumbersBrowseRequest {
+  /** Represents the number type of the offering. */
+  phoneNumberType: PhoneNumberType;
+  /** Capabilities of a phone number. */
+  capabilities?: PhoneNumberCapabilitiesRequest;
+  /** Represents the assignment type of the offering. Also known as the use case. */
+  assignmentType?: PhoneNumberAssignmentType;
+  /** The phone number prefix to match. If specified, the search will be limited to phone numbers that start with the any of the given prefixes. */
+  phoneNumberPrefixes?: string[];
+}
+
+/** Capabilities of a phone number. */
+export interface PhoneNumberCapabilitiesRequest {
```

Comment:
Was this not generated before?

Reply:
I was there previously, however it was called 'PhoneNumberCapabilities' and is present on line 115.